### DISCUSSION TITLE
App 1716740 未包含游戏配置，无法自动入库。

### DISCUSSION THREAD

**Discussion by lpx2223** [pjy612/SteamManifestCache#3] [discussion]
请问提示这个是什么意思？这个是下载您发的里面自带有1716740_config.vdf 。但是cmd运行exe后输入代码就提示未包含游戏配置，试过很多个都提示错误，是我哪里操作错误了吗？还有请问您的群是多少？

**Comment by pjy612** [comment]
请问使用的是什么版本，如果用的 1.7.1 的话，那个版本写了个bug 不识别本地的 vdf
